UITests/Responses/TransferMethod/Individual/TransferMethodConfigurationKeysResponse.json

```diff
@@ -199,17 +199,9 @@
                                         {
                                             "code": "BANK_ACCOUNT",
                                             "name": "Bank Account",
-                                            "processingTime": "1-2 Business days",
+                                            "processingTime": "",
```

Comment:
You will want to have two options.  One where the we do not have the processing time and another where do not have the fee node.  Looks like this.
```
{
                    "code": "GB",
                    "name": "United Kingdom",
                    "currencies": {
                        "nodes": [
                            {
                                "code": "EUR",
                                "name": "Euro",
                                "transferMethodTypes": {
                                    "nodes": [
                                        {
                                            "code": "BANK_ACCOUNT",
                                            "name": "Bank Account",
                                            "fees": {
                                                "nodes": [
                                                    {
                                                        "country": "GB",
                                                        "currency": "EUR",
                                                        "transferMethodType": "BANK_ACCOUNT",
                                                        "value": "1.50",
                                                        "feeRateType": "FLAT"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                }
                            },
                            {
                                "code": "GBP",
                                "name": "British Pound",
                                "transferMethodTypes": {
                                    "nodes": [
                                        {
                                            "code": "BANK_ACCOUNT",
                                            "name": "Bank Account",
                                            "processingTime": "1-2 Business days"
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                }
```